photon-server/src/main/java/org/photonvision/vision/target/TargetModel.java

```diff
@@ -81,27 +82,27 @@ public static TargetModel get2020Target() {
     }
 
     public static TargetModel get2020TargetInnerPort() {
-        return get2020Target(2d * 12d + 5.25); // Inches, TODO switch to meters
+        return get2020Target(Units.inchesToMeters(2d * 12d + 5.25));
```

Comment:
Can we maybe just convert the values themselves instead?

Reply:
Wdym?

Reply:
Instead of 2 inches, 0.0508 meters, and for all values in this file. It's just very cluttered with all the conversion calls